locale/fi/LC_MESSAGES/django.po

```diff
@@ -2889,7 +2889,15 @@ msgstr "Tehtävässä on huoltokatko, ja sisältö on piilotettu opiskelijoilta.
 msgid "Unfortunately this exercise is currently under maintenance."
 msgstr "Valitettavasti tässä tehtävässä on juuri nyt huoltokatko."
 
-#: exercise/views.py:181
+#: exercise/views.py:155
+msgid ""
```

Comment:
I think `python manage.py makemessages` is supposed to add a comment `#, python-brace-format` here because the string contains the `{error}` format variable.